ts/session/messages/outgoing/controlMessage/ExpirationTimerUpdateMessage.ts

```diff
@@ -30,13 +30,6 @@ export class ExpirationTimerUpdateMessage extends DataMessage {
     this.syncTarget = params.syncTarget ? PubKey.cast(params.syncTarget).key : undefined;
   }
```

Comment:
This removes backwards compat with legacy disppearing messages. You happy to start doing this now?